website/blog/2020-06-27-lithium.md

```diff
@@ -0,0 +1,503 @@
+can be done manually by clicking on the pop up whenever new script is opened or
+selecting automatic import at the start, which will import any open scripts.
+
+It's important to note that this is an early approach to the support, which
+might still require some improvements both on the Metals and Ammonite side. You
+learn more about the new feature or comment the in the
+[blogpost](https://contributors.scala-lang.org/t/ammonite-support-in-metals/4306)
+on the Scala contributors forum.
+
+Most notable features of Ammonite scripts are:
+
+- importing other scripts, Metals provide completions to select existing files
+  based on the filesystem information
+
+```scala
+//main.sc
+import $file.hello-world
+val greeting = hello-world.greeting
+
+//hello-world.sc
+val greeting = "Hello world!"
+```
+
+- importing dependecies
+
+```scala
+import $ivy.`com.lihaoyi::scalatags:0.7.0`
+import scalatags.Text.all._
+
+val rendered = div("Moo").render
+```
+
+## Worksheet dependency support
+
+In addition to Ammonite support, which will be useful to anyone interested in
+Scala scripting, we've also improved the support for Metals Scala worksheets.
+Thanks to the efforts of [olafurpg](https://github.com/olafurpg), we now are
+able to provide importing dependencies inside your Scala worksheets with
+completions and diagnostics working right from the bat.
+
+![worksheet-imports](https://i.imgur.com/qXgdNWM.gif)
+
+You can add them using the following syntax:
+
+```
+$dep.`organisation::artifact:version`
+```
+
+for example:
+
+```
+$dep.`com.lihaoyi::scalatags:0.9.0`
+```
+
+`::` is the same as `%%` in sbt, which will append the current scala binary
+version to the artifact name.
+
+## New project provider
+
+Creating new Scala projects is usually a bit of work to get started, which is a
+problem for both beginners and experts alike. Normally you don't want a lot of
+time to figure out the best setup for starting with a new technology or the
+language itself.
+
+There are great tools like [giter8](http://www.foundweekends.org/giter8/), which
+is used in the `sbt new` command, that enables people to use already existing
+Scala templates found on Github such as https://github.com/scala/scala-seed.g8.
+The problem is when you don't use sbt or you don't know the exact template you
+want to use.
+
+In this version we added an additional command called `New Scala Project` that
+enables users to reuse the same mechanism in comfort of your own UI. Firstly,
+you can choose from existing templates, either curated by the Metals team, ones
+existing on the giter8
+[wiki page](https://github.com/foundweekends/giter8/wiki/giter8-templates) or
+alternatively an entirely custom templates. Then you can choose the exact
+location for the new project and the name. Afterwards, Metals will run giter8
+itself and ofer to open the new project.
+
+![new-project](https://user-images.githubusercontent.com/3807253/81842954-2d1b3680-954d-11ea-934a-96fe53cafeb9.gif)
```

Comment:
Should I make a smaller gif here? I wanted to show the entire workspace properly, but it might be too big.

Reply:
I don't think it looks too bad, but if you are worried about it, just cutting in half and only showing the top should be fine as well.